dfsclient/src/main/java/org/hpccsystems/dfs/client/DataPartition.java

```diff
@@ -415,6 +416,8 @@ public String toString()
                 DataPartition new_dp = new DataPartition(clusterremapper.reviseIPs(dfuparts[i].getCopies()), copyPaths, dfuparts[i].getPartIndex(),
                         dfuparts.length, clusterremapper.revisePort(null), clusterremapper.getUsesSSLConnection(null), filter, fileAccessBlob,
                         fileType);
+                new_dp.isTLK = dfuparts[i].getKind() == DFUFilePartWrapper.FilePartKind.TLK;
```

Comment:
Per offline discussion, istlk() should be used based on this change:
https://github.com/hpcc-systems/hpcc4j/pull/436